packages/lib/src/dimension-mapper/SlicingSlider.module.css

```diff
@@ -29,38 +32,34 @@
 
 .track {
   flex: 1;
-  margin: calc(var(--thumb-height) / 2) 0;
-  border-left: 2px solid var(--secondary-dark);
-  border-radius: 2px;
+  margin: calc(var(--thumb-size) / 2) 0;
+  border-left: var(--track-width) solid var(--track-color);
+  border-radius: var(--track-width);
 }
 
 .thumb {
   display: flex;
   align-items: center;
   justify-content: center;
-  padding: 0.125rem;
-  background-color: var(--primary-bg);
-  border: 2px solid var(--secondary);
-  border-radius: 1.5em;
-  height: var(--thumb-height);
-  min-width: 1.5rem;
-  font-size: 0.625rem;
+  padding: 0.125rem 0.25rem;
+  background-color: var(--h5w-slicingSlider-thumb--bgColor, aliceblue);
+  border: 2px solid var(--h5w-slicingSlider-thumb--borderColor, cornflowerblue);
+  border-radius: calc(var(--thumb-size) / 2);
+  height: var(--thumb-size);
+  min-width: var(--thumb-size);
   cursor: grab;
 }
 
-.thumb:global(.active) {
-  border-color: var(--secondary-dark);
-}
-
-.thumb:active {
-  cursor: none;
+.thumbValue {
+  font-size: 0.625em;
 }
```

Comment:
The `active` class was no longer being applied (probably a regression from a long time ago). I chose not to fix it because `react-slider` adds this class when the thumb has focus, so the class remains after you've interacted with the thumb (until you click somewhere else) and it looks weird if the thumb keeps a different color after you're done interacting with it...